models/nft/nft_wash_trades.sql

```diff
@@ -0,0 +1,112 @@
+{{ config(
+        alias ='wash_trades',
+        partition_by='block_date',
+        materialized='incremental',
+        file_format = 'delta',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["hildobby"]\') }}',
+        unique_key = ['unique_trade_id']
+)
+}}
+
+SELECT
+    nftt.blockchain
+    , nftt.project
+    , nftt.version
+    , nftt.nft_contract_address
+    , nftt.token_id
+    , nftt.token_standard
+    , nftt.trade_category
+    , nftt.buyer
+    , nftt.seller
+    , nftt.project_contract_address
+    , nftt.aggregator_name
+    , nftt.aggregator_address
+    , nftt.tx_from
+    , nftt.tx_to
+    , nftt.block_time
+    , date_trunc('day', nftt.block_time) AS block_date
+    , nftt.block_number
+    , nftt.tx_hash
+    , nftt.unique_trade_id
+    , filter_funding_buyer.first_funded_by AS buyer_first_funded_by
+    , filter_funding_seller.first_funded_by AS seller_first_funded_by
+    , count(1)
```

Comment:
the commented out CASE statements below still need resolved, and this count can be removed. that was only used as reference for me to share where the query stood.

you'll need these case output columns for final spell i imagine